recipe/conda_build_config.yaml

```diff
@@ -0,0 +1 @@
+minimum_jaxlib_version: 0.4.14
```

Comment:
I think I prefer this value in the recipe so we remember to update it in newer versions.

Reply:
Except that is exactly what was _forgotten_ previously.

I moved it to `conda_build_config.yaml` so it would only appear in a single place, but could also be specifically pinned in the test section.

`import jax` checks the version compatibility, so if you forget to update it the package tests will fail.

Reply:
> import jax checks the version compatibility, so if you forget to update it the package tests will fail.

It did not fail before though. Any idea what happened there?

PS: I'm one of the maintainers and I always double check before merging. If I merged that one it was my mistake.

Reply:
> It did not fail before though. Any idea what happened there?
the tests previously didn't constrain to the minimum supported version - conda solves in the test with the highest version possible, and they pass.

In a real environment with other constraints you may be able to get the latest `jax` but only an out of date `jaxlib` - which is what happened to me (and prompted this fix).